torch_ttnn/passes/lowering/to_tt_pass.py

```diff
@@ -634,16 +634,27 @@ def rewrite_node(node):
                     return g.call_function(ttnn.reshape, args=(args[0], output_size))
                 return None
 
-            if node.target == torch.ops.aten.transpose.int:
-                dim0 = args[1]
-                dim1 = args[2]
+            if node.target in [torch.ops.aten.transpose.int, torch.ops.aten.t.default]:
                 output_size = node.meta["val"].size()
                 rank = len(output_size)
+                if node.target == torch.ops.aten.t.default:
+                    assert rank >= 0 and rank <= 2, "Input tensor can only be 0D, 1D or 2D"
```

Comment:
instead of asserting, should we fallback?

Reply:
Given the pytorch spec, this shouldn't be possible: https://pytorch.org/docs/stable/generated/torch.t.html

"Expects input to be <= 2-D tensor and transposes dimensions 0 and 1."

So I think it makes sense to use assert here

Reply:
Thank you!